Engine/Graphics/Indoor.h

```diff
@@ -739,83 +735,6 @@ bool PointInPolyOutdoor(int a1, int a2, int a3, struct ODMFace *face,
 
 #pragma once
 
-/*  165 */
```

Comment:
I have been working on a partial reinstatement of the blv portal frustum culling. Please leave this for now.

Reply:
Is just BspRenderer_PortalViewportData enough? Or you want me to leave the whole block including BspRenderer as is?

Reply:
Could you leave the whole block as is and then I will cleanup this on my branch.

Reply:
Done.